fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -577,7 +578,7 @@ protected void addLinkAndOptionsHttpHeaders(final FedoraResource resource) {
      * @param relation the relation string.
      * @return the string version of the link.
      */
-    private static String buildLink(final String linkUri, final String relation) {
+    public static final String buildLink(final String linkUri, final String relation) {
```

Comment:
In general, making methods `final` creates less flexible code. Additionally, in this case, `static` removes any functional effects of being `final`.
I would suggest not making this method `final`.